lib/src/geo/crs.dart

```diff
@@ -1,6 +1,6 @@
 import 'dart:math' as math;
+import 'dart:math';
```

Comment:
Use only one import for the same library.

Reply:
Nice catch 👍